src/organization/organization.reducer.js

```diff
@@ -7,63 +19,65 @@ const initialState = {
   isPendingOrg: false,
   isPendingRepos: false,
   isPendingMembers: false,
-  error: '',
+  organizationError: '',
```

Comment:
Created one error for organization, another for repositories and another for members.